README.md

```diff
@@ -141,21 +161,20 @@ Next, you'll need to add the Watson Discovery credentials to the .env file.
   # your own before starting the app.
 
   # Watson Discovery
-  DISCOVERY_URL=<add_discovery_url>
-  DISCOVERY_ENVIRONMENT_ID=<add_discovery_environment_id>
-  DISCOVERY_COLLECTION_ID=<add_discovery_collection_id>
+  DISCOVERY_PROJECT_ID=<add_discovery_project_id>
   DISCOVERY_APIKEY=<add_discovery_apikey>
+  DISCOVERY_URL=<add_discovery_url>
   ```
 
 ## 5. Run scripts to generate data
 
-The provided scripts can be used to generate sample data for a facticious company, Named "Acme Coffee". The company offers 5 types of coffee, which is sold in grocery stores.
+The provided scripts can be used to generate sample data for a factitious company, Named "Acme Coffee". The company offers 5 types of coffee, which is sold in grocery stores.
 
-The scripts build the following data, which will be output to local .csv files:
+The scripts build the following data, which will be output to local CSV files:
 
-* Reviews - this contains all of the reviews for each of the products. Includes rating and sentiment score.
-* Products - this is information about each of our products. Includes product_id, name, and unit price.
-* Keywords - this contains keywords generated from product reviews.
+* Reviews [`data/out-reviews.csv`] - this contains all of the reviews for each of the products. Includes rating and sentiment score.
+* Products [`data/out-products.csv`] - this is information about each of our products. Includes product_id, name, and unit price.
+* Keywords [`data/out-keywords.csv`] - this contains keywords generated from product reviews.
```

Comment:
The links don't work.  I think for these 3 links you want the format [data/out-keywords.csv](data/out-keywords.csv] so it will show like a link with the path/file.csv name you want to show, but the clicking on it also takes you to the file.